uaclient/api/u/pro/security/cves/v1.py

```diff
@@ -282,141 +252,119 @@ def _post_process_vulnerability_info(
         return vulnerability_info
 
 
+def cve_status_match_options(cve, options) -> bool:
+    is_fixable = cve.get("fix_version") and cve.get("fix_origin")
+
+    if options.unfixable and is_fixable:
+        return False
+    elif options.fixable and not is_fixable:
+        return False
+    return True
+
+
 def vulnerabilities(
     options: CVEVulnerabilitiesOptions,
-) -> CVEVulnerabilitiesResult:
+) -> PackageVulnerabilitiesResult:
     return _vulnerabilities(options, UAConfig())
```

Comment:
I think @renanrodrigo is right that these need renamed to align with our convention of the function name matching the module name.
Also really the options and result objects should be named similarly.
```suggestion
def cves(
    options: CVEsOptions,
) -> CVEsResult:
    return _cves(options, UAConfig())
```